website/docs/api/misc/eslint-plugin/no-html-links.md

```diff
@@ -0,0 +1,24 @@
+---
+slug: /api/misc/@docusaurus/eslint-plugin/no-html-links
+---
+
+# no-html-links
+
+Ensure that docusaurus `<Link>` is used instead of `<a>` tags.
+
+## Rule Details {#details}
+
+Examples of **incorrect** code for this rule:
+
+```html
+<a href="/page">go to page!</a>
+```
+
+Examples of **correct** code for this rule:
+
+```js
+//
+import Link from '@docusaurus/Link';
+
+<Link to="/page">go to page!</Link>;
+```
```

Comment:
Do you think it would suffice with adding an extra line of correct/incorrect usage or something more descriptive? 

`<a href="https://wwww,twitter.com/docusaurus" target="_blank">Twitter</a>`

`<Link to="https://www.twitter.com/docusaurus">Twitter</Link>`

And maybe we should link ( :drum: ) to the Link docs from here as well?

Reply:
Yes, just add more examples of good/bad usage

And a link to Link 👍 why not, users could find this helpful